Binary files oaf-0.6.7/.configure.in.swp and oaf-0.6.7-new/.configure.in.swp differ
diff -Nur oaf-0.6.7/configure.in oaf-0.6.7-new/configure.in
--- oaf-0.6.7/configure.in	Wed Oct 24 20:42:54 2001
+++ oaf-0.6.7-new/configure.in	Mon Nov  5 12:14:28 2001
@@ -37,7 +37,7 @@
 
 dnl internationalization support
 ALL_LINGUAS="az da de el es fi fr ga gl hu it ja ko nl nn no pl pt pt_BR ro ru sk sl sv ta tr uk zh_TW"
-AM_GNU_OAF_GETTEXT
+AM_GNU_GETTEXT
 
 
 dnl Checks for libraries.
